janusgraph-berkeleyje/src/test/resources/log4j.properties

```diff
@@ -16,12 +16,12 @@ log4j.appender.A1=org.apache.log4j.FileAppender
 log4j.appender.A1.File=target/test.log
 log4j.appender.A1.Threshold=ALL
 log4j.appender.A1.layout=org.apache.log4j.PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
```

Comment:
Why did you change the log pattern of Berkeleyje?

Reply:
I apologize. I was trying to isolate some test failures locally and was trying to find the timestamps for each test. I reverted this change now.